src/index.ts

```diff
@@ -621,26 +617,32 @@ export default class Purchases {
    */
   public static INTRO_ELIGIBILITY_STATUS = INTRO_ELIGIBILITY_STATUS;
 
+  /**
+   * Enum of all error codes the SDK produces. 
+   * @readonly
+   * @enum {string}
+   */
+  public static PurchasesErrorCode = PurchasesErrorCode;
```

Comment:
This I don't love - I'm doing Pascal case, which seems to be the standard recommended in a couple of places (like [Microsoft's styleguide](https://www.typescriptlang.org/docs/handbook/enums.html), [this other styleguide](https://basarat.gitbook.io/typescript/styleguide#enum) and [Google's styleguide](https://google.github.io/styleguide/tsguide.html#identifiers)), and which honestly looks nicer to me. 

However, the rest of the enums are defined in upper case. 
So it's either the outdated, but consistent notation, or the new and nicer one, but inconsistent. 

I actually think it's probably better to do upper case for now, and then in the next major maybe move them all to lower together. 

Thoughts?

Reply:
Let's do what you suggest

Reply:
c80e54d86d2e49126b465893ec51339b23cd47a1